applications/FluidDynamicsApplication/custom_processes/shock_capturing_physics_based_process.cpp

```diff
@@ -86,7 +86,11 @@ namespace Kratos
             "shock_sensor" : true,
             "shear_sensor" : true,
             "thermal_sensor" : true,
-            "thermally_coupled_formulation" : true
+            "thermally_coupled_formulation" : true,
+            "artificial_Bulk_viscosity_constant": 1.5,
```

Comment:
```suggestion
            "artificial_bulk_viscosity_constant": 1.5,
```
Our json formating always uses snake_case.